pixl_imaging/src/pixl_imaging/_orthanc.py

```diff
@@ -264,5 +290,51 @@ async def send_study_to_anon(self, resource_id: str) -> Any:
         )
 
         logger.debug("Successfully triggered c-store of study to anon modality: {}", resource_id)
-
         return str(response["ID"])
+
+
+class PIXLAnonOrthanc(Orthanc):
+    """Orthanc Anon connection."""
+
+    def __init__(self) -> None:
+        super().__init__(
+            url=config("ORTHANC_ANON_URL"),
+            username=config("ORTHANC_ANON_USERNAME"),
+            password=config("ORTHANC_ANON_PASSWORD"),
+            http_timeout=config("PIXL_QUERY_TIMEOUT", default=10, cast=int),
+            dicom_timeout=config("PIXL_DICOM_TRANSFER_TIMEOUT", default=240, cast=int),
+            aet=config("ORTHANC_ANON_AE_TITLE"),
+        )
+
+        self.autoroute_to_endpoint = config(
+            "ORTHANC_AUTOROUTE_ANON_TO_ENDPOINT", default=False, cast=bool
+        )
+
+    async def import_study_from_raw(self, orthanc_raw: PIXLRawOrthanc, resource_id: str) -> Any:
+        """
+        Notify Orthanc Anon of a study to pull from Raw
+
+        - Query Orthanc Raw for the study
+        - Send the StudyInstanceUID and the query ID to Orthanc Anon
+        """
+        orthanc_raw_study_info = await orthanc_raw.query_local_study(study_id=resource_id)
+        study_uid = orthanc_raw_study_info["MainDicomTags"]["StudyInstanceUID"]
+
+        data = {
+            "Level": "Study",
+            "Query": {
+                "StudyInstanceUID": study_uid,
+            },
+        }
+        query_id = await self.query_remote(modality="PIXL-Raw", data=data)
+        if query_id is None:
+            logger.info(f"No unique study found in Orthanc Raw with StudyInstanceUID: {study_uid}.")
+            return
+
+        logger.info("Importing study {} from raw to anon", study_uid)
+
+        # Don't wait for Orthanc Anon to finish processing the study
+        with contextlib.suppress(TimeoutError):
+            await self._post(
+                "/import-from-raw", data={"StudyInstanceUID": study_uid, "QueryID": query_id}
+            )
```

Comment:
what's the best way to not wait for a response from Orthanc Anon?

Reply:
we could just not await the response?

Reply:
hmm it seems like if we don't await the response, then `self._post` is never called. We can add the call to `self._post` inside `asyncio.create_task()` to avoiding awaiting it, but a timeout error still seems to be raised in the imaging API

Reply:
ah intersting, okay that makes sense as long as we've documented what we're doing there

Reply:
yeah good point - I've added a clarifying comment